src/astarte-client/client.ts

```diff
@@ -447,20 +448,38 @@ class AstarteClient {
   }
 
   async getBlocks(): Promise<AstarteBlock[]> {
+    const mockedBlocks = mocks.blocks as AstarteBlockDTO[];
```

Comment:
Same as above, those are not meant to be mock ups, those will be fully functional